website/content/commands/force-leave.mdx

```diff
@@ -44,6 +44,9 @@ are not supported from commands, but may be from the corresponding HTTP endpoint
 
 Usage: `consul force-leave [options] node`
 
+`node` can [contain URL-invalid characters](/commands#accepts-url-invalid-characters),
```

Comment:
As far as I can tell, we don't have a place in most of our command docs pages for a CLI parameter (like `node`), only for the `[options]`. I'm open to suggestions on a better way to include information about this capability in the CLI docs.

Reply:
Just to back it up a second, does `node` in this section literally mean that you must include the word `node`? Or do we mean `<node_name>`?

Reply:
Changed to ...

---

## Usage

Usage: `consul force-leave [options] <node_name>`

#### Positional Arguments

- `node_name` - The name of the node to be forced into the `left` state. If the name contains URL-invalid characters, [provide the raw name without URL-encoding](/commands#arguments-with-url-invalid-characters).

#### API Options

...